### ISSUE TITLE
Increasing max length of sentence

### ISSUE THREAD

**Issue by youngkyung** [vpegasus/seq2seq_chatbot#1]
I tried to modify your codes but when the max length of sentence is increased, the codes make an InvalidArgumentError which says the dimension of logits and labels are not same. Please check this problem that users can use the codes with larger max length of sentence. Thank you!
